portal/src/graphql/portal/LocalizationConfigurationScreen.tsx

```diff
@@ -900,22 +820,23 @@ const LocalizationConfigurationScreen: React.VFC =
     const [selectedLanguage, setSelectedLanguage] =
       useState<LanguageTag | null>(null);
 
-    const config = useAppConfigForm({
-      appID,
-      constructFormState,
-      constructConfig,
-    });
+    // const config = useAppConfigForm({
+    //   appID,
+    //   constructFormState,
+    //   constructConfig,
+    // });
```

Comment:
It seems this comment can be removed

Reply:
I fixed up in [0df1192](https://github.com/authgear/authgear-server/pull/4331/commits/0df1192f2f05a60926d02ba490dbc74e699d49e2)